contracts/modules/claims/ClaimsReward.sol

```diff
@@ -89,15 +89,16 @@ contract ClaimsReward is IClaimsReward, LegacyMasterAware {
       _changeClaimStatusCA(claimId, coverid, status);
     } else if (status >= 1 && status <= 5) {
       _changeClaimStatusMV(claimId, coverid, status);
-    } else if (status == 12) {// when current status is "Claim Accepted Payout Pending"
+    } else if (status == 12) {
 
-      bool payoutSucceeded = attemptClaimPayout(coverid);
+      // status == 12 is "Claim Accepted Payout Pending"
+      // technically should be unreachable after partial claim implementation
+      // since we revert if the payout fails
 
-      if (payoutSucceeded) {
-        c1.setClaimStatus(claimId, 14);
-      } else {
-        c1.setClaimStatus(claimId, 12);
-      }
+      attemptClaimPayout(coverid);
+
+      // 14 = payout succeeded
+      c1.setClaimStatus(claimId, 14);
```

Comment:
so this change assumes the payout always succeeds and you bypass status 12 (Claim Accepted Payout Pending) all together. I think this is fine for the purpose of FTX claims. 

side question: can `attemptClaimPayout` not be successful for any other reason than not enough tokens in the pool?

Reply:
Was there another expected wave of claims that we expect before v2 launch?

Reply:
> can attemptClaimPayout not be successful for any other reason than not enough tokens in the pool?
yes, if:
- target address rejects eth transfers (contracts without receive/fallback)
- token to be transfered has blacklisted the pool or the recipient

I do not expect it to generally fail with reasons other than insufficients funds.